src/resources/postcss/tickets-registration-page.pcss

```diff
@@ -186,6 +186,31 @@ body.page-tribe-attendee-registration {
 	padding: 0 0 30px 0;
 }
 
+.tribe-block__tickets__item__attendee__fields__success {
+	color: rgb(24, 217, 0);
+	display: none;
+	padding: 0 0 30px 0;
+}
+
+.tribe-block__tickets__item__attendee__fields__loader {
+	background: rgba( 255, 255, 255, .7 );
+	display: none;
+	position: absolute;
```

Comment:
sorta nit-picky, but in products, we alphabetize properties (not going to repeat this for all the pcss :wink:)